src/apps/expenses/components/Transactions.js

```diff
@@ -178,7 +178,7 @@ class Transactions extends React.Component {
                 collective={collective}
                 {...transaction}
                 isRefund={Boolean(transaction.refundTransaction)}
-                canRefund={LoggedInUser && LoggedInUser.isRoot()}
+                canRefund={LoggedInUser && (LoggedInUser.isRoot() || LoggedInUser.isHostAdmin(collective))}
```

Comment:
@flickz this is wrong, your checking if the collective is a host and if the authenticated user is admin of the collective.

What you want is checking if the user is the admin of the collective's host.